pydoctor/astutils.py

```diff
@@ -737,3 +740,178 @@ class Precedence(object):
 
 del _op_data, _index, _precedence_data, _symbol_data, _deprecated
 # This was part of the astor library for Python AST manipulation.
+
+
+# Part of the sphinx.pycode.parser module.
+# Copyright 2007-2020 by the Sphinx team, see AUTHORS.
```

Comment:
Could this code be isolated to a separate file to make license provenance at least a *little* less confusing?